src/ui/components/shared/LoadingScreen.tsx

```diff
@@ -55,8 +56,11 @@ function LoadingScreen({
 
   return (
     <LoadingScreenTemplate showTips={true}>
-      <span>{stalled ? "This is taking longer than usual..." : waitingForMessage}</span>
-    </LoadingScreenTemplate>
+    <div className="flex text-xs">
+      <span>{waitingForMessage}</span>
+      <span className="ml-2">{stalled ? "This is taking longer than usual..." : ""}</span>
```

Comment:
I'm surprised we want a small margin on the left here? Are we aligning it with something?